about.html

```diff
@@ -0,0 +1,171 @@
+<!DOCTYPE html>
+<html lang="en">
+<head>
+    <meta charset="UTF-8">
+    <meta http-equiv="X-UA-Compatible" content="IE=edge">
+    <meta name="viewport" content="width=device-width, initial-scale=1.0">
+    <title>About- OV Gaming Summit</title>
+    <link rel="stylesheet" href="style.css">
+</head>
+<body>
+    <div id="overlay"></div>
+    <div id="mobile-menu" class="mobile-main-menu flex-col">
+        <img src="asset/fire.png" alt="">
+        <ul class="flex-col">
+            <li class="flex-row"><a href="index.html" class="menu-item">Home</a></li>
+            <li class="flex-row"><a href="about.html" id="link1" class="menu-item">About</a></li>
+            <li class="flex-row"><a href="#main-section" class="menu-item">Program</a></li>
+            <li class="flex-row"><a href="#" class="menu-item">Speakers</a></li>
+            <li class="flex-row"><a href="#sponsors" class="menu-item">Sponsors</a></li>
+            <li class="flex-row"><a href="#" class="menu-item">News</a></li>
+        </ul>
+    </div>
+    <header>
+        <div id="top-bar">
+            <ul>
+                <li>
+                    <a href="#"><i class="fa-brands fa-facebook-f"></i></a>
+                </li>
+                <li>
+                    <a href="#"><i class="fa-brands fa-twitter"></i></a>
+                </li>
```

Comment:
- [ ] Please try to implement the things in a professional and well mannered. As your previous code reviewer *pointed* out that you have the `Twitter` and `Facebook` links but these icon links are not visible on both the *home-page* and *about-page*, but it seems like you have not fixed the issue yet. Kindly either remove it *OR* try to fix the issue so that it will appear on both pages, leaving unnecessary code is not good at all, it creates confusion, maybe make an extra request to `font-awesome` which indeed increases the page load time a little bit.

#### Look at here

![image](https://github.com/over-geek/capstone-project-2/assets/66476329/09418fa6-7732-416e-9681-5411c9cdd567)